src/middlewared/middlewared/client/client.py

```diff
@@ -198,7 +198,7 @@ def result(self):
                 trace={
                     'class': job['exc_info']['type'],
                     'formatted': job['exception'],
-                    'repr': job['exc_info']['repr'],
+                    'repr': job['exc_info'].get('repr'),
```

Comment:
Let's use the last line of the traceback when this is not available then
```suggestion
                    'repr': job['exc_info'].get('repr', job['exception'].splitlines()[-1]),
```

Reply:
Great idea.